guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>For this functionality to work in ArcGIS Online, Azure library packages for Python (Azure SDK for Python - azure-storage-blob: 12.1&lt;= version &lt;=12.8) needs to be pre-installed. Refer here for more information <a href="https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-install" target="_blank">https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-install</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>